docs/process/release.md

```diff
@@ -18,7 +18,7 @@ TBD
 ## Merging PRs
 
 The list of PRs which are currently ready to merge (approved with passing status checks) can
-be found with [this search](https://github.com/angular/angular-cli/pulls?q=is%3Apr+is%3Aopen+review%3Aapproved+status%3Asuccess).
+be found with [this search](https://github.com/angular/angular-cli/pulls?q=is%3Apr+is%3Aopen+review%3Aapproved+status%3Asuccess+label%3A%22PR+action%3A+merge%22).
```

Comment:
I wonder if `review:approved status:success` are necessary. There are cases where those might not be true and something is still ready for a merge, but with a caretaker comment. 

If we have a label that explicitly says "caretaker, please merge this", then that should be the source of truth. If there's a PR with the label that isn't ready to be merged, the caretaker should remove the label and say why it's being removed.

Having the meaning of the label be conditional on other checks seems to me is an opportunity for some PRs to be forgotten.

Reply:
That's a good point, if the author makes a mistake it is good to show it to the caretaker who can make the intelligent decision on how to handle it.

The only other concern I see is that a caretaker might not manually verify that a PR is approved and passing CI and accidentally merge something which hasn't passed those checks. It seems that GitHub has the "Rebase and merge" button in bright red in this case, so that would be a tough mistake to make and probably not worth worrying about.

Removed `review:approved` and `status:success`.